docs/gen2_Tutorial.md

```diff
@@ -429,6 +429,73 @@ and clean them up.
 
 If the cluster which which is being deleted is a cluster created using patch files, the controller first cleans the git repo where the respective patch files of the cluster are present and then it destroys all the related ArgoCD applications and clean them up.
 
+## Enabling Cluster Autoscaler in the workload cluster
+To create a gen2 cluster with autoscaler, we need:
+
+- bundle pointing to the bundle/capi-cluster-autoscaler in the arlon repository.
+- dynamic profile that contains the above bundle.
+- a basecluster manifest which has the CAPI annotations for min and max nodes set ( as a part of `preparegit` or manually add it ).
+- run `arlon cluster create` with the repo-path pointing to the basecluster manifest described in the step above, set the profile to  be the one created in step 2 and pass the `autoscaler` flag.
+
+### Bundle creation
+
+Register a dynamic bundle pointing to the bundles/capi-cluster-autoscaler in the Arlon repo.
+
+The capi-cluster-autoscaler bundle requires the name of the cluster, so that it knows what namespace in the management cluster to scan for CAPI resources. To enable the cluster-autoscaler bundle, add one more parameter during cluster creation: `srcType`. This is the ArgoCD-defined application source type (Helm, Kustomize, Directory).
```

Comment:
The first sentence is confusing. Why does the user need to know this? Does the user need to worry about the name of the cluster? If this is an internal implementation detail, let's delete that sentence and only keep the fact that srcType is required.

Reply:
Not really, it's an implementation detail. I will remove this and just let the user-specific part here